src/scriptworker/context.py

```diff
@@ -169,7 +169,10 @@ def temp_credentials(self):
     @temp_credentials.setter
     def temp_credentials(self, credentials):
         self._temp_credentials = credentials
-        self.temp_queue = self.create_queue(self.temp_credentials)
+        if credentials is not None:
+            self.temp_queue = self.create_queue(self.temp_credentials)
+        else:
+            self.temp_queue = None
```

Comment:
Out of curiosity, when would we reach this `else` case ?

Reply:
Here? https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/context.py#L91